test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DisasterRecovery/BackupTests.cs

```diff
@@ -94,5 +98,90 @@ public void CreateBackupTest()
             // Clean up the database
             testDb.Cleanup();
         }
+
+        /// <summary>
+        /// Test creating backup with advanced options set.
+        /// </summary>
+        [Fact]
+        public void CreateBackupWithAdvancedOptionsTest()
+        {
+            string databaseName = "testbackup_" + new Random().Next(10000000, 99999999);
+            SqlTestDb testDb = SqlTestDb.CreateNew(TestServerType.OnPrem, false, databaseName);
+            string certificateName = "backupcertificate" + new Random().Next(10000000, 99999999);
+            string createCertificateQuery = string.Format(CreateCertificateQueryFormat, certificateName);
+            string cleanupCertificateQuery = string.Format(CleanupCertificateQueryFormat, certificateName);
+
+            // create master key and certificate
+            testDb.RunQuery(createCertificateQuery);
+            var liveConnection = LiveConnectionHelper.InitLiveConnectionInfo(databaseName);
+
+            // Initialize backup service
+            DatabaseTaskHelper helper = AdminService.CreateDatabaseTaskHelper(liveConnection.ConnectionInfo, databaseExists: true);
+            SqlConnection sqlConn = DisasterRecoveryService.GetSqlConnection(liveConnection.ConnectionInfo);
+
+            // Get default backup path
+            BackupConfigInfo backupConfigInfo = DisasterRecoveryService.Instance.GetBackupConfigInfo(helper.DataContainer, sqlConn, sqlConn.Database);
+            string backupPath = backupConfigInfo.DefaultBackupFolder + "\\" + databaseName + ".bak";
```

Comment:
>"\\" [](start = 71, length = 4)

you should prefer Path.Combine to manually building paths...or at least use Path.PathSeparator instead of "\\"